solr/core/src/java/org/apache/solr/servlet/ServletUtils.java

```diff
@@ -143,9 +143,6 @@ static boolean excludedPath(
       for (Pattern p : excludePatterns) {
         Matcher matcher = p.matcher(requestPath);
         if (matcher.lookingAt()) {
-          if (chain != null) {
-            chain.doFilter(request, response);
```

Comment:
wow that was a sneaky side-effect of what looked like a simple boolean method

Reply:
Ah actually looking there, there is a change that probably should be made, since patterns are being set (but now ignored).